### ISSUE TITLE
Update Docker Compose v2

### ISSUE THREAD

**Issue by LINDFJ25477-pki** [actions/virtual-environments#5349]
### Tool name

Docker Compose v2

### Tool license

?

### Add or update?

- [ ] Add
- [X] Update

### Desired version

&gt; 2.3.3

### Virtual environments affected

- [ ] Ubuntu 18.04
- [X] Ubuntu 20.04
- [ ] macOS 10.15
- [ ] macOS 11
- [ ] macOS 12
- [ ] Windows Server 2019
- [ ] Windows Server 2022

**Comment by al-cheb** [comment]
Docker Compose v2 2.3.3 is already preinstalled on Ubuntu Server 20.04 - https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md

**Comment by LINDFJ25477-pki** [comment]
Sorry, we need version after 2.3.3. We are running into the [issue described here](https://github.com/actions/virtual-environments/pull/5335).

**Comment by LINDFJ25477-pki** [comment]
@al-cheb  Sorry, we need version after 2.3.3. We are running into the [issue described here](https://github.com/docker/compose/issues/9304).

**Comment by al-cheb** [comment]
@LINDFJ25477-pki , docker compose v2 has not been yet updated in the repo - https://packages.microsoft.com/ubuntu/20.04/prod/pool/main/m/moby-compose/ . We will try to ask maintainers to update it and let you know.

**Comment by al-cheb** [comment]
@LINDFJ25477-pki, The new version of moby-compose has been released and now it's 2.4.1. You could install it in runtime:

```
sudo apt-get update
sudo apt-get upgrade moby-compose -y
docker compose version
```

**Comment by al-cheb** [comment]
@LINDFJ25477-pki, The new Ubuntu Server 20.04 image has been deployed with `Docker Compose v2 2.4.1+azure-1` on board.
